pkg/apis/servicecatalog/types.go

```diff
@@ -127,10 +127,9 @@ type ServiceClass struct {
 	Plans         []ServicePlan
 	PlanUpdatable bool // Do we support this?
 
-	// OSB-specific
-	// OSBGUID is the identity of this object for use with the OSB API.
+	// ExternalID is the identity of this object for use with the OSB API.
 	// Immutable.
-	OSBGUID string
+	ExternalID string
```

Comment:
I can try to undo it, but it was difficult enough changing only what I needed to that I went and hit it with the big hammer of global search and replace.